sources/packages/backend/apps/api/src/route-controllers/note/note.aest.controller.ts

```diff
@@ -56,7 +57,7 @@ export class NoteAESTController extends BaseController {
   @Get("student/:studentId")
   async getStudentDetails(
     @Param("studentId", ParseIntPipe) studentId: number,
-    @Query("noteType") noteType: NoteType,
+    @Query("noteType", new ParseEnumQueryPipe(NoteType)) noteType: NoteType,
```

Comment:
`noteType` is probably nullable here, right?

Reply:
Yes that was the bug, when 'All notes' is selected.